userspace/libscap/scap_savefile.c

```diff
@@ -1108,6 +1113,7 @@ static int32_t scap_read_proclist(scap_t *handle, gzFile f, uint32_t block_lengt
 		case PL_BLOCK_TYPE_V5:
 			break;
 		case PL_BLOCK_TYPE_V6:
+		case PL_BLOCK_TYPE_V7:

```

Comment:
Can you also initialize `tinfo->exepath[0] = 0` or similar in this function? Otherwise, if you read from a trace file < V7, you don't know what garbage the array might be filled with, and that might get propagated to sinsp.